### ISSUE TITLE
Accessibility-flavored React components make your design system delicious! -  KM

### ISSUE THREAD

**Issue by nickttng** [A11YChi/meetups-and-events#19]
**Why:** As an attendee, I want to learn how to build an inclusive design system.

- **Title:** Accessibility-flavored React components make your design system delicious!
- **Synopsis:** Design systems are a popular way for teams to flavor their design and development workflow. However, an often-missing ingredient in many design systems is a focus on accessibility best practices — especially when component libraries are involved. In this talk, we’ll take a look at how you can mix some commonly-used components with the ingredients of accessibility. Pair this with best practices guidance in your documentation, and you’ll have the recipe for a delectably inclusive design system.
- **Primary audience:** Developers and/or engineers, especially front-end, and designers
- **Secondary audience:** ??? Product owners, anyone who works on digital and web products, etc.
- **When** ~Potentially, June or July~ Fall/Winter 2020 - Tentative: October 28th

---

## ToDo

**30 days before the event**
- [x] Co-org to reach out to presenter
- [x] Have a kick-off conversation
- [x] Coordinate with co-orgs for date availability
- [x] Coordinate with the presenter for next steps
  - [x] Get synopsis 
  - [x] Get bio
  - [x] Set potential dates
  - [x] Share 'Accessible Presentation' guideline
- [x] Secure ACS captioner
- [x] Create event page/URL

**15 days before the event**
- [x] Announce through channels (MTLS)
  - [x] Mailing list
  - [x] Twitter
  - [ ] LinkedIn
  - [x] Web-A11Y Slack
- [x] Share event page/URL with presenter
- [x] If feasible, request (rough) slides from presenter for _initial_ accessibility check

**1-5 days before the event**
- [ ] Reminders through channels (MTLS)
  - [ ] Mailing list
  - [ ] Twitter
  - [ ] LinkedIn
  - [ ] Web-A11Y Slack
- [x] Follow up with presenter for being online (5-10 before official "online") and any needs
- [x] Update housekeeping slides
- [x]  Presenter's slides
  - [x] Request slides from presenter for _final_ accessibility check
  - [ ] Send presenter's slides to ACS
  - [x] Upload to SpeakerDeck

**Day of event**
- [x] Co-orgs are in video meeting 5-10 minutes to discuss any needs/issues before the official "online" for attendees
- [x] Coordinate with presenter
- [x] Announcements and talk

**1-5 days after the event**
- [x] Post-announcement, YouTube share, and thanks (MTLS)
  - [x] Shoutout for presenter
- [x] Finalize transcript
  - [x] Get transcript from ACS
  - [x] Edit transcript and push to GitHub transcript repo

**Post-event (based on team bandwidth)**
- [x] Add transcript to YouTube video talk

**Comment by nickttng** [comment]
This event has concluded and the [video](https://www.youtube.com/watch?v=NKyjqqfEi0E) is posted on YouTube with CC. We  are closing this issue.
